src/index.html

```diff
@@ -1,18 +1,65 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <!-- <div class="card__top"> -->
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="#"
+      />
+      <!-- </div> -->
+      <div class="card__description card__description--margin--top">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code card__code--margin--top">Product code: 195434</div>
```

Comment:
```suggestion
      <div class="card__code">Product code: 195434</div>
```
you don't need separate modifier just for margin-top, add this property to element style, and fix it everywhere